src/app/routes/contentRoutes.js

```diff
@@ -82,17 +82,13 @@ module.exports = (app) => {
                 }
             })
         )
-        app.all('/content/questionset/v1/copy/:id',
+    app.all('/content/questionset/v1/copy/:id',
```

Comment:
please add logger

Reply:
@princegupta1131 
Please avoid using `app.all`, specify what is the request type for the API
Ex:
app.post -> for post method
app.get -> for get method
CC: @rajeevsathish